includes/admin/class-wc-stripe-express-checkouts-controller.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Automattic\WooCommerce\Admin\PageController;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Admin page for UPE Customize Express Checkouts.
+ *
+ * @since 5.4.1
+ */
+class WC_Stripe_Express_Checkouts_Controller {
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'admin_scripts' ] );
+		add_action( 'wc_stripe_gateway_admin_options_wrapper', [ $this, 'admin_options' ] );
+	}
+
+	/**
+	 * Load admin scripts.
+	 */
+	public function admin_scripts() {
+		// Webpack generates an assets file containing a dependencies array for our built JS file.
+		$script_path       = 'build/express_checkouts_customizer.js';
+		$script_asset_path = WC_STRIPE_PLUGIN_PATH . '/build/express_checkouts_customizer.asset.php';
+		$script_url        = plugins_url( $script_path, WC_STRIPE_MAIN_FILE );
+		$script_asset      = file_exists( $script_asset_path )
+			? require $script_asset_path
+			: [
+				'dependencies' => [],
+				'version'      => WC_STRIPE_VERSION,
+			];
+		wp_register_script(
```

Comment:
Should the stylesheet file from `build/` also be imported here or is it enough to just import the script?

Reply:
Great catch! Missed that. :+1